packages/datetime/src/dateInput.tsx

```diff
@@ -221,24 +227,24 @@ export class DateInput extends AbstractComponent<IDateInputProps, IDateInputStat
         return this.props.invalidDateMessage;
     }
 
-    private validAndInRange(value: moment.Moment) {
-        return value.isValid() && this.dateIsInRange(value);
+    private isMomentValidAndInRange(value: moment.Moment) {
+        return isMomentValidAndInRange(value, this.props.minDate, this.props.maxDate);
     }
 
-    private isNull(value: moment.Moment) {
-        return value.parsingFlags().nullInput;
+    private isMomentInRange(value: moment.Moment) {
+        return isMomentInRange(value, this.props.minDate, this.props.maxDate);
```

Comment:
Yeah, this just makes the lines shorter. `, this.props.minDate, this.props.maxDate);` is hella long and is always used with the same parameters thus far within this class. Happy to not do this though.